### PR TITLE
Responsive demo

### LINKED ISSUE #46: Design a better demo page
It needs to be responsive (use some micro responsive CSS framework).
It should also start the FP process when button is clicked, not on page load.

### PR DESCRIPTION
**Pull request by ericorruption**
Fixes #46.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,23 +12,85 @@
...
-    fp.get(function(result) {
-      var d2 = new Date();
-      var timeString = "Time took to calculate the fingerprint: " + (d2 - d1) + "ms";
-      console.log(timeString);
-      console.log(result);
-      document.getElementById("fp").innerHTML = result;
-      document.getElementById("time").innerHTML = timeString;
+    var btn = document.getElementById('btn');
+
+    btn.addEventListener('click', function setFingerPrint() {
```
Comment (Valve) [inline index.html]: Will this work on IE8?

**Comment by ericorruption** [comment]
`addEventListener` won't  work, but `console` doesn't work either.

**Comment by Valve** [comment]
oh, I guess it's fine then for a demo page, thanks a lot!

**Comment by Valve** [comment]
@ericorruption , would it be too much of me to ask you to please add the svg spinner, which is active when the FP is being calculated?  The reason is because on mobile the FP process can take up to 3 seconds. If you don't have time or just don't feel like it, no worries, you've helped the project a lot :)

**Comment by ericorruption** [comment]
sure, I'll look into that. Do I make another PR for this change?

2015-07-07 13:30 GMT-03:00 Valentin Vasilyev <notifications@github.com>:

> @ericorruption <https://github.com/ericorruption> , would it be too much
> of me to ask you to please add the svg spinner, which is active when the FP
> is being calculated? The reason is because on mobile the FP process can
> take up to 3 seconds. If you don't have time or just don't feel like it, no
> worries, you've helped the project a lot :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Valve/fingerprintjs2/pull/48#issuecomment-119261148>.
>



-- 
Eric Quanz
Designer turned front-end developer
ericquanz.com / +55 51 9325-6643

**Comment by Valve** [comment]
Yes, please

**Pull request merged**